docs/interpolation.rst

```diff
@@ -79,11 +71,75 @@ For components represented by discretized PDFs (PSF and EDISP tables) these clas
     Nucl. Instrum. Methods Phys. Res. A 771, 39-48. https://doi.org/10.1016/j.nima.2014.10.033
 
 
-Usage Example
--------------
+Using Estimator Classes
+-----------------------
+
+Usage of the estimator classes is simple. 
+As an example, consider CTA's Prod5 IRFs [CTA+21]_, they can be downloaded manually or by executing 
+``download_irfs.py`` in ``pyirf's`` root directory, which downloads them to ``.../pyirf/irfs/``.
+The estimator classes can simply be used by first creating an instance of the respective class with all 
+relevant information and then using the object's ``__call__`` interface the obtain results for a specific 
+target point.
+As the energy dispersion represents one of the discretized PDF components, one can use the 
+``MomentMorphInterpolator`` for interpolation and the ``DiscretePDFNearestNeighborSearcher`` 
+for extrapolation.
+
+.. code-block:: python
+
+   import numpy as np
+
+   from gammapy.irf import load_irf_dict_from_file
+   from glob import glob
+   from pyirf.interpolation import (
+      EnergyDispersionEstimator, 
+      MomentMorphInterpolator,
+      DiscretePDFNearestNeighborSearcher
+   )
+
+   # Load IRF data, replace path with actual path
+   PROD5_IRF_PATH = "pyirf/irfs/*.fits.gz"
+
+   irfs = [load_irf_dict_from_file(path) for path in sorted(glob(PROD5_IRF_PATH))]
+
+   edisps = np.array([irf["edisp"].data for irf in irfs])
```

Comment:
not important for edisp, but maybe because it's an example use `.quantity` instead of `.data`?